### DISCUSSION TITLE
Straight line (beeline) route profile (works offline)

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#446] [Q&A] [answered]
Straight line (beeline) profile will be available in all routing services.

It will work instantly and offline: setting a straight line on a route section, _no route calculation will be needed_.

This will be possible as a result of the new:
- https://github.com/devemux86/cruiser/discussions/142

**Comment by umpfmpf** [comment]
Is it also possible after I have created a tour from several sections with a direct line to change and calculate it completely as a route?

**Reply by devemux86** [reply]
@umpfmpf do you mean to remove the straight lines and calculate the route in these sections?
This is also possible.

**Reply by umpfmpf** [reply]
Section for section or also all at once?

**Reply by devemux86** [reply]
Straight line profile is set by the user on route sections between waypoints.

- [Intelligent route calculation](https://github.com/devemux86/cruiser/discussions/142) will calculate each section separately.

- [Manual route calculation](https://github.com/devemux86/cruiser/discussions/449) will calculate the whole route after completing all changes.

**Comment by devemux86** [comment]
Straight line (beeline) profile in all routing services (works offline) is available for testing in:
- https://github.com/devemux86/cruiser/discussions/451

**Comment by devemux86** [comment, marked as answer]
Straight line (beeline) profile in all routing services (works offline) is available in [Cruiser 3.0.41](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
